Task 1/Train.rb

```diff
@@ -26,70 +26,88 @@
 - Возвращать предыдущую станцию, текущую, следующую, на основе маршрута.
 =end
 
-class Train  
-  attr_accessor :speed, :number, :type
-
-  attr_reader :num_of_carriages 
+class Train
+  attr_reader :number, :type, :num_of_carriages, :speed, :route, :station_index
 
   def initialize(number, type, num_of_carriages)
     @number = number
     @type = type
     @num_of_carriages = num_of_carriages
-    
-    @speed = 0    
+
+    @speed = 0
     @route = nil
-    @@register = nil
+    @station_index = nil
   end
 
-  def speed_up(v)
-    @speed = v
+  def speed_up(add_speed)
+    @speed = @speed + add_speed
```

Comment:
```suggestion
    @speed += add_speed
```
можно так записать, то что ты реализовал в 43 строке